projects/1.16/assets/materialis/materialis/lang/zh_cn.json

```
@@ -0,0 +1,259 @@
+  "modifier.materialis.freezing.flavor": "像冰一样冷！",
+  "modifier.materialis.galvanized": "电镀",
+  "modifier.materialis.galvanized.description": "用 2 个升级槽换取 1 个能力槽",
+  "modifier.materialis.galvanized.flavor": "不锈钢！",
+  "modifier.materialis.half_life": "半衰期",
+  "modifier.materialis.half_life.3": "半衰期 III",
+  "modifier.materialis.half_life.4": "半衰期 IV",
+  "modifier.materialis.half_life.description": "耐久度高的时候会更快地消耗耐久，耐久度低的时候则消耗得更慢",
+  "modifier.materialis.half_life.flavor": "闻到了灰烬的味道",
+  "modifier.materialis.inertia": "惯性",
+  "modifier.materialis.inertia.description": "攻击速度降低，但攻击伤害提高",
+  "modifier.materialis.inertia.flavor": "物体的性质就是如此",
+  "modifier.materialis.neptunes_blessing": "海王星的祝福",
+  "modifier.materialis.neptunes_blessing.description": "工具在水下的采掘速度提升，并增加对水栖生物造成的伤害",
+  "modifier.materialis.neptunes_blessing.flavor": "保持水分",
+  "modifier.materialis.nocturnal": "夜曲",
+  "modifier.materialis.nocturnal.description": "工具在夜间的采掘速度提升，午夜效果最强",
+  "modifier.materialis.nocturnal.flavor": "别睡觉了！",
+  "modifier.materialis.old_timer": "旧钟",
+  "modifier.materialis.old_timer.description": "工具的耐久度越高，修复消耗的材料越多",
+  "modifier.materialis.old_timer.flavor": "锈桶！",
+  "modifier.materialis.otherworldly": "彼岸",
+  "modifier.materialis.otherworldly.description": "允许采掘彼岸矿物",
+  "modifier.materialis.otherworldly.flavor": "来自另一个存在位面！",
+  "modifier.materialis.overweight": "超重",
+  "modifier.materialis.overweight.description": "采掘速度和攻击速度降低，但会获得更多的黏液覆层",
+  "modifier.materialis.overweight.flavor": "超大容量！",
+  "modifier.materialis.polished": "抛光",
+  "modifier.materialis.polished.description": "耐久度越高，攻击伤害越高",
+  "modifier.materialis.polished.flavor": "我能从里面看到我的脸！",
+  "modifier.materialis.psicho_digger": "Psi挖掘者",
+  "modifier.materialis.psicho_digger.description": "拥有的Psi越多，采掘速度越快",
+  "modifier.materialis.psicho_digger.flavor": "跑，跑远点",
+  "modifier.materialis.psicho_killer": "Psi杀手",
+  "modifier.materialis.psicho_killer.description": "拥有的Psi越多，攻击伤害越高",
+  "modifier.materialis.psicho_killer.flavor": "那是什么？",
+  "modifier.materialis.quenching": "淬火",
+  "modifier.materialis.quenching.description": "扑灭目标身上的火，并根据灼烧程度造成额外伤害",
+  "modifier.materialis.quenching.flavor": "不可燃！",
+  "modifier.materialis.reaping": "收割",
+  "modifier.materialis.reaping.description": "击杀亡灵生物会破坏他们的肉体并结晶灵魂，似乎对镰刀情有独钟",
+  "modifier.materialis.reaping.flavor": "灵魂收割者",
+  "modifier.materialis.refueling": "火上浇油",
+  "modifier.materialis.refueling.description": "击中的目标如果处于燃烧状态，则使其燃烧时间加长",
+  "modifier.materialis.refueling.flavor": "加大火力",
+  "modifier.materialis.residual_light": "余光",
+  "modifier.materialis.residual_light.description": "对目标施加发光效果，并且采掘方块会制造光源",
+  "modifier.materialis.residual_light.flavor": "锻光",
+  "modifier.materialis.runed": "符文",
+  "modifier.materialis.runed.description": "工具闪烁着彩色的魔法光芒",
+  "modifier.materialis.runed.flavor": "能量流符文！",
+  "modifier.materialis.short_sighted": "目光短浅",
+  "modifier.materialis.short_sighted.description": "减少接触距离",
```

|Feedback Priority|
|---|
|⚪️ Pebble / Non-blocking but requires future action|

```suggestion
  "modifier.materialis.short_sighted.description": "减少触及距离",
```